docs/pages/billing/invoices-and-receipts.mdx

```diff
@@ -0,0 +1,73 @@
+description: Learn how to view your account's payment history, download invoices and receipts, and request a refund for a charge.
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+**Receipts** in the Expo dashboard provide information about an account's payment history and access to invoices and receipts. It also provides information on payment dates, payment status, and the total amount for that payment. You can also request a refund for a charge if you believe it has been made in error.
+
+> **Note**: You can only access the **Receipts** if you have [Owner or Admin access](/accounts/working-together/#manage-access) to your account.
+
+## Receipts
+
+To view your account's payment history, click [**Receipts**](https://expo.dev/settings/receipts) in the navigation menu under **Account settings** or **Organization settings**.
+
+For example, an [Organization account's](/accounts/account-types/#organizations) receipts are shown below:
+
+<ImageSpotlight
+  alt="Receipts page in the Expo dashboard."
+  src="/static/images/billing/receipts-01.png"
+/>
+
+### Download and view an invoice
+
+To download and view an invoice for a billing period, go to **Receipts** and:
+
+- Click the **Date** for the billing period corresponding to the invoice. You will be navigated to a page hosted by Stripe. As an example, the March 22, 2024 invoice below links to this page:
+
+<ImageSpotlight
+  alt="Download invoice shown in the Stripe dashboard."
+  src="/static/images/billing/receipts-02.png"
+  style={{ maxWidth: 380 }}
+/>
+
+- Click **Download invoice**. You will receive a PDF copy of the invoice.
+
+### Download and view a receipt
+
+To download and view a receipt for a billing period, go to **Receipts** and:
+
+- Click the **Date** for the billing period corresponding to the receipt. You will be navigated to the appropriate receipt hosted by Stripe. As an example, the March 22, 2024 receipt below links to this page:
+
+<ImageSpotlight
+  alt="Download receipt shown in the Stripe dashboard."
+  src="/static/images/billing/receipts-02.png"
+  style={{ maxWidth: 380 }}
+/>
+
+- Click on **Download receipt.** You will receive a PDF copy of the receipt.
+
+### Request a refund
+
+You can request a refund directly from the **Receipts** page. The approval process is manual and our team investigates any errors before providing a refund.
+
+To request a refund:
+
+- Next to a receipt, click the a three-dot menu and then click **Request Refund:**
+
+<ImageSpotlight
+  alt="Receipts page in the Expo dashboard."
+  src="/static/images/billing/receipts-04.png"
+/>
+
+- Fill the **Request a refund** form with details for the refund and click **Continue**:
+
+<ImageSpotlight
+  alt="Download invoice shown in the Stripe dashboard."
+  src="/static/images/billing/receipts-05.png"
+  style={{ maxWidth: 320 }}
+/>
+
+- The refund request goes through our vetting process before it is granted. When it is granted, the amount is credited to your account.
```

Comment:
```suggestion
- Our billing team receives and reviews refund requests. Once a refund is approved, the amount is credited to your payment method. Refunds typically take 5-10 business days to fully process.
```
Especially want to clarify "account" (e.g., refunds wouldn't credit to your Expo account)

Reply:
Thanks for catching this!